app/controllers/exercises_controller.rb

```diff
@@ -30,7 +30,9 @@ def start!
   end
 
   def set_default_content
+    @file_content = @exercise.files_content_for(current_user).except('content')
```

Comment:
I'd remove the `except('content')` from here, and hide this implementation detail in the `files_content_for` method

Reply:
done